.github/workflows/ci-app-accessibility-tests.yml

```diff
@@ -0,0 +1,66 @@
+name: CI App Accessibility Tests
+
+on:
+  workflow_call:
+    inputs:
+      service_endpoint_url:
+        required: true
+        type: string
+
+jobs:
+  build:
+    name: Accessibility
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./app
+
+    env:
+      NODE_VERSION: 20
+      LOCKFILE_PATH: ./app/package-lock.json
+      PACKAGE_MANAGER: npm
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache-dependency-path: ${{ env.LOCKFILE_PATH }}
+          cache: ${{ env.PACKAGE_MANAGER }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Create screenshots directory
+        run: mkdir -p screenshots-output
+
+      - name: Set service endpoint URL
+        run: echo "SERVICE_ENDPOINT_URL=\"${{ inputs.service_endpoint_url }}\"" >> "$GITHUB_ENV"
+        shell: bash
+
+      - name: Update pa11y-ci config for desktop
+        run: |
+          jq ".defaults.actions += [\"navigate to ${SERVICE_ENDPOINT_URL}\"]" .pa11yci-desktop.json > .pa11yci-desktop-temp.json
+          mv .pa11yci-desktop-temp.json .pa11yci-desktop.json
+
+      - name: Update pa11y-ci config for mobile
+        run: |
+          jq ".defaults.actions += [\"navigate to ${SERVICE_ENDPOINT_URL}\"]" .pa11yci-mobile.json > .pa11yci-mobile-temp.json
+          mv .pa11yci-mobile-temp.json .pa11yci-mobile.json
+
+      - name: Run pa11y-ci
+        run: |
+          set -e # Ensure the script fails if any command fails
+          npm run test:pa11y-desktop
+          npm run test:pa11y-mobile
+          echo "pa11y-ci tests finished."
+
+      - name: Upload screenshots to artifacts
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: screenshots
+          path: ./app/screenshots-output
```

Comment:
question: what do we do with this / how do we view the screenshots?

Reply:
You can see them in the artifacts on the job run - https://github.com/navapbc/template-application-nextjs/actions/runs/9555816763

Just a sanity check on what's actually being observed.